packages/algob/sample-project/test/sample-test.js

```diff
@@ -0,0 +1,72 @@
+import { getProgram } from '@algorand-builder/algob';
+import { Runtime, StoreAccount, types } from '@algorand-builder/runtime';
+const { assert } = require('chai');
+
+const minBalance = BigInt(1e6);
+const masterBalance = BigInt(10e6);
+const amount = BigInt(1e6);
+
+describe('Sample Test', function () {
+  let master;
+  let fundReceiver;
+
+  let runtime;
+  const feeCheckProgram = getProgram('fee-check.teal');
+
+  this.beforeEach(async function () {
+    master = new StoreAccount(masterBalance);
+    fundReceiver = new StoreAccount(minBalance);
+    runtime = new Runtime([master, fundReceiver]);
+  });
+
+  function syncAccounts () {
+    master = runtime.getAccount(master.address);
+    fundReceiver = runtime.getAccount(fundReceiver.address);
+  }
+
+  it('Should not fail because txn fees is equal to or greater than 10000 microAlgos', () => {
+    const fees = 10000;
+    syncAccounts();
+    assert.deepEqual(fundReceiver.balance(), minBalance);
+    assert.deepEqual(master.balance(), masterBalance);
+    const lsig = runtime.getLogicSig(feeCheckProgram);
+    lsig.sign(master.account.sk);
+    runtime.executeTx({
+      type: types.TransactionType.TransferAlgo,
+      sign: types.SignType.LogicSignature,
+      lsig: lsig,
+      fromAccount: master.account,
+      toAccountAddr: fundReceiver.address,
+      amountMicroAlgos: amount,
+      payFlags: { totalFee: fees }
+    });
+    syncAccounts();
+    assert.deepEqual(fundReceiver.balance(), minBalance + amount);
+    assert.deepEqual(master.balance(), masterBalance - amount - BigInt(fees));
+  });
+
+  it('Should fail because txn fees is less than 10000 microAlgos', () => {
+    const fees = 1000;
+    syncAccounts();
+    assert.deepEqual(fundReceiver.balance(), minBalance);
+    assert.deepEqual(master.balance(), masterBalance);
+    const lsig = runtime.getLogicSig(feeCheckProgram);
+    lsig.sign(master.account.sk);
+    try {
+      runtime.executeTx({
+        type: types.TransactionType.TransferAlgo,
+        sign: types.SignType.LogicSignature,
+        lsig: lsig,
+        fromAccount: master.account,
+        toAccountAddr: fundReceiver.address,
+        amountMicroAlgos: amount,
+        payFlags: { totalFee: fees }
+      });
+    } catch (error) {
+      console.log(error);
+    }
+    syncAccounts();
+    assert.notEqual(fundReceiver.balance(), minBalance + amount);
```

Comment:
Please add assert for initial balance here.
assert.equal(fundReceiver.balance(), initialBalance)
same for master

Reply:
okay.